api/presenter/shared.go

```diff
@@ -184,9 +184,9 @@ func emptySliceIfNil(m []string) []string {
 	return m
 }
 
-func formatTimestamp(t *time.Time) *string {
+func formatTimestamp(t *time.Time) *time.Time {
```

Comment:
This function is no longer formatting timestamps, it converts a `time.Time` instance to UTC time. Therefore, I believe that `toUTC` or similar should be a more appropriate name.